elastic/apm-server
# ci(release minor): add tag::list entry for minor release in the changelog

## Motivation/summary

Automate https://github.com/elastic/apm-server/pull/14023 for the new minor version.

## Checklist




For functional changes, consider:
- Is it observable through the addition of either **logging** or **metrics**?
- Is its use being published in **telemetry** to enable product improvement?
- Have system tests been added to avoid regression?

## How to test these changes

```bash
$ gh pr checkout 14025
$ export GITHUB_TOKEN=$(gh auth token)
$ make minor-release RELEASE_VERSION=8.16.0 PROJECT_OWNER=v1v BASE_BRANCH=8.x
```

<img width="607" alt="image" src="https://github.com/user-attachments/assets/592e1284-3403-48b1-9d80-0f7fee6b5dd9">

See https://github.com/v1v/apm-server/pull/48


## Related issues
